qcodes/instrument_drivers/QuTech/IVVI.py

```diff
@@ -260,17 +338,15 @@ def read(self, message_len=None):
 
         while bytes_in_buffer < message_len:
             t1 = time.time()
-            time.sleep(.05)
+            time.sleep(.025)
```

Comment:
> I already changed this but haven't been able to commit it to the new repo ye

@giulioungaretti , I was wondering, these PR's are based on branches but we don't have write access to the the new QCoDeS repo. How can @Rubenknex make changes to this branch in order to get it to merge?